src/gluonts/model/r_forecast/_predictor.py

```diff
@@ -49,57 +50,46 @@
 R -e 'install.packages(c("forecast", "nnfor"), repos="https://cloud.r-project.org")'
 """
 
-SAMPLE_FORECAST_METHODS = ["ets", "arima"]
-QUANTILE_FORECAST_METHODS = ["tbats", "thetaf", "stlar"]
-POINT_FORECAST_METHODS = ["croston", "mlp"]
-SUPPORTED_METHODS = (
-    SAMPLE_FORECAST_METHODS
-    + QUANTILE_FORECAST_METHODS
-    + POINT_FORECAST_METHODS
-)
-
 
 class RForecastPredictor(RepresentablePredictor):
     """
     Wrapper for calling the `R forecast package
     <http://pkg.robjhyndman.com/forecast/>`_.
 
-    The `RForecastPredictor` is a thin wrapper for calling the R forecast
-    package.  In order to use it you need to install R and run::
+    The `RForecastPredictor` is a thin wrapper for calling R `forecast` and `hts` packages.
+    In order to use it you need to install R and run::
 
         pip install 'rpy2>=2.9.*,<3.*'
         R -e 'install.packages(c("forecast", "nnfor"), repos="https://cloud.r-project.org")'
+        R -e 'install.packages(c("hts"), repos="https://cloud.r-project.org")'
```

Comment:
Maybe we can avoid mentioning what is required here, since further changes will require updating the docstring again. How about mentioning that “specific R packages are required, depending on which wrapper one needs to run”?